<!-- MHonArc v2.1.0 -->
<!--X-Subject: [pgsql&#45;jp 9258] Re: 6.5の JDBCにおける接続時の URL指定 -->
<!--X-From: "N.Ogiso" <ogiso@use&#45;net.co.jp> -->
<!--X-Date: Mon, 5 Jul 1999 19:18:00 +0900 -->
<!--X-Message-Id: 199907051016.TAA05843@ns.use&#45;net.co.jp -->
<!--X-ContentType: text/plain -->
<!--X-Head-End-->
<HTML>
<HEAD>
<TITLE>[pgsql-jp 9258] Re: 6.5の JDBCにおける接続時の URL指定 </TITLE>
<LINK REV="made" HREF="mailto:ogiso@use-net.co.jp">
</HEAD>
<BODY BGCOLOR="#ffffff">
<!--X-Body-Begin-->
<!--X-User-Header-->
<!--X-User-Header-End-->
<!--X-TopPNI-->
<HR>
[<A HREF="msg00061.html">Date Prev</A>][<A HREF="msg00063.html">Date Next</A>][<A HREF="msg00069.html">Thread Prev</A>][<A HREF="msg00065.html">Thread Next</A>][<A HREF="index.html#00062">Date Index</A>][<A HREF="threads.html#00062">Thread Index</A>]
<!--X-TopPNI-End-->
<!--X-MsgBody-->
<!--X-Subject-Header-Begin-->
<H1>[pgsql-jp 9258] Re: 6.5の JDBCにおける接続時の URL指定</H1>
<HR>
<!--X-Subject-Header-End-->
<UL>
<LI><em>From</em>: "N.Ogiso" &lt;<A HREF="mailto:ogiso@use-net.co.jp">ogiso@use-net.co.jp</A>&gt;</LI>
<LI><em>Date</em>: Mon, 5 Jul 1999 19:18:00 +0900</LI>
</UL>
<!--X-Head-Body-Sep-Begin-->
<HR>
<!--X-Head-Body-Sep-End-->
<!--X-Body-of-Message-->
<PRE>
こんにちは。

----------
&gt; 差出人 : Kazumasa Gotoh &lt;kgotoh@cic-kk.co.jp&gt;
&gt; 宛先 : pgsql-jp@sra.co.jp
&gt; 件名 : [pgsql-jp 9253] Re: 6.5の JDBCにおける接続時の URL指定
&gt; 送信日時 : 1999年7月5日 18:46
&gt; 
&gt; 
snip...
&gt; 
&gt; しかし、SQLExpetion が発生するたびに落ちてしまうのではどうしようも
&gt; ないので、上記の問題は何とかしないといけないとは思います…
&gt; 
&gt; リソースファイルの直し方がおかしいのでしょうか？ ちなみに変なホスト名を
&gt; セットして実行すると、
&gt; 
&gt;   DB アクセス例外発生: Something unusual has occured to cause the driver
to fail. Please report this exception: java.lang.IllegalArgumentException:
Argument # &gt; Arg length
&gt; 
&gt; となります。catch は
&gt; 
&gt;         } catch(SQLException e) {
&gt;             System.out.println("DB アクセス例外発生: " + e.getMessage());
&gt;         }
&gt; 
&gt; としているのですが、上記のようなメッセージで正常なのでしょうか？
&gt; 
&gt; いや… これで正常のような気がしてきました。とすると、エラく不親切な
&gt; メッセージなような… (^^;
&gt; 

getMessage() で取得できているので、正常なメッセージなんで
しょうね。# たしかに不親切 (^^;;
これはリソースファイルを修正したら SQLException で catch できる
ようになったということでしょうか。
# やっぱりバグだったのかな？

---
おぎそ
</PRE>

<!--X-Body-of-Message-End-->
<!--X-MsgBody-End-->
<!--X-Follow-Ups-->
<HR>
<UL><LI><STRONG>Follow-Ups</STRONG>:
<UL>
<LI><STRONG><A NAME="00065" HREF="msg00065.html">[pgsql-jp 9261] Re: 6.5の JDBCにおける接続時の URL指定</A></STRONG>
<UL><LI><EM>From:</EM> Kazumasa Gotoh &lt;kgotoh@cic-kk.co.jp&gt;</LI></UL></LI>
</UL></LI></UL>
<!--X-Follow-Ups-End-->
<!--X-References-->
<!--X-References-End-->
<!--X-BotPNI-->
<UL>
<LI>Prev by Date:
<STRONG><A HREF="msg00061.html">[pgsql-jp 9257] Re: JDBC Driver for 6.5</A></STRONG>
</LI>
<LI>Next by Date:
<STRONG><A HREF="msg00063.html">[pgsql-jp 9259] Re: can't exec one more postmaster with "semget failed" Error</A></STRONG>
</LI>
<LI>Prev by thread:
<STRONG><A HREF="msg00069.html">[pgsql-jp 9265] Re: 6.5の JDBCにおける接続時の URL指定</A></STRONG>
</LI>
<LI>Next by thread:
<STRONG><A HREF="msg00065.html">[pgsql-jp 9261] Re: 6.5の JDBCにおける接続時の URL指定</A></STRONG>
</LI>
<LI>Index(es):
<UL>
<LI><A HREF="index.html#00062"><STRONG>Date</STRONG></A></LI>
<LI><A HREF="threads.html#00062"><STRONG>Thread</STRONG></A></LI>
</UL>
</LI>
</UL>

<!--X-BotPNI-End-->
<!--X-User-Footer-->
<!--X-User-Footer-End-->
</BODY>
</HTML>
